### PR TITLE
security: Adding page to about section

### PR DESCRIPTION
**Pull request by geek**
Content comes from @evilpacket in PR https://github.com/joyent/node-website/pull/60

### CODE REVIEW

**Comment by evilpacket** [comment]
@geek thanks for making this go. I apologize for not being able to touch up the various suggestions. This is great.

**Comment by geek** [comment]
@evilpacket I just want to make sure you can say that node at least has a security page when you present at node summit!

File: doc/about/security/index.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "template": "doc/about.html", "title": "Security" }
```
Comment (robertkowalski) [inline doc/about/security/index.json]: missing newline

**Comment by robertkowalski** [comment]
small issue: please add the link also to: https://github.com/joyent/node-website/blob/master/doc/about/advisory-board/template.html#L37-L46 - the advisory board has an own template

**Comment by robertkowalski** [comment]
+1 after adressing the minor nits I found, great work everybody!

**Comment by geek** [comment]
@robertkowalski all fixed up

**Comment by robertkowalski** [comment]
+1

**Comment by robertkowalski** [comment]
merge it! :)

File: doc/about/security/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+will be acknowledged.
+
+
+## Disclosure Policy
+
+Here is the security disclosure policy for Node.js
+
+- The security report is received and is assigned a primary handler. This person will coordinate the fix and release
+process. The problem is confirmed and a list of all affected versions is determined. Code is audited to find any
+potential similar problems. Fixes are prepared for all releases which are still under maintenance. These fixes are not
```
Comment (misterdjules) [inline doc/about/security/index.md:28]: Regarding

>  Fixes are prepared for all releases which are still under maintenance

Are the releases under maintenance documented somewhere? [It is confusing for some users at least](https://groups.google.com/forum/#!topic/nodejs/f0dzje1VQhc), so I think we should probably have that information easily available on the website.

**Comment by misterdjules** [comment]
@evilpacket @geek @robertkowalski Thank you!

**Comment by misterdjules** [comment]
@geek @robertkowalski I don't see the security page online. When is the next upload/deploy scheduled for the website?

**Comment by geek** [comment]
@misterdjules trying to determine if I need the core teams approval first.  @tjfontaine do you know if I can just upload the new security page?

**Comment by tjfontaine** [comment]
I think this is fine, the content is fairly non-controversial and the truth -- I say go for it. In general my opinion is you don't need to seek core team approval unless you think it's changing process for the core team. But we will discuss on the call tomorrow.

**Comment by geek** [comment]
The site is updated, the security page is published: http://nodejs.org/about/security/

**Comment by misterdjules** [comment]
@geek @robertkowalski @evilpacket Thanks again :+1:

**Pull request merged**